### PR TITLE
Disable refresh control while searching for tags

### LINKED ISSUE #8478: Tags: refreshing an empty tag search results in the "No Tags Yet" screen
Steps to reproduce:

1. Start from a site that has a lot of tags saved (there are 176 tags on the site I tested).
1. Go to WPiOS > Settings > Tags.
1. Search for a non-existent tag.
1. While the empty search is still showing, wwipe down to refresh the page.

<strong>Result:</strong> I see a message that says "No Tags Yet Would you like to create one?" even though the site has several tags. I expected that either the empty search result would remain or that I the whole tags page would refresh and I would see the entire tag list.

### PR DESCRIPTION
**Pull request by bummytime**
Title has it. This PR simply disables the refresh control while the user is searching for tags in WPiOS > Settings > Tags.

Fixes #8478 

To test: See the issue for repro details.

@elibud would you mind when you have a moment?

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Blog/SiteManagement/SiteTagsViewController.swift [hunk 1]
```diff
@@ -101,13 +108,15 @@ final class SiteTagsViewController: UITableViewController {
...
         tableView.register(nibName, forCellReuseIdentifier: TableConstants.cellIdentifier)
-        setupRefreshControl()
+        activateRefreshControl()
     }
 
-    private func setupRefreshControl() {
-        let control = UIRefreshControl()
-        control.addTarget(self, action: #selector(refreshTags), for: .valueChanged)
-        refreshControl = control
+    private func activateRefreshControl() {
```
Comment (elibud) [inline WordPress/Classes/ViewRelated/Blog/SiteManagement/SiteTagsViewController.swift]: I don't really like using a lazy load variable to initialize another variable, it's a bit hard to follow. We could do something like `setupRefreshControl` and `deactivateRefreshControl` and setup can initialize the variable if it's nil, what do you think?

**Comment by elibud** [comment]
I left a small comment but I think the idea on this PR should work, however I have never been able to replicate the original issue, nor in the simulator nor in the device, so I don't trust that I'm the right person to double check that is fixed. @koke would you mind taking this one?

File: WordPress/Classes/ViewRelated/Blog/SiteManagement/SiteTagsViewController.swift (hunk 1 above)
Comment (bummytime) [reply WordPress/Classes/ViewRelated/Blog/SiteManagement/SiteTagsViewController.swift]: @elibud Works for me!

**Comment by bummytime** [comment]
@elibud I pushed your suggested changes. @koke FWIW, I was able to originally repro this only in the sim a handful of times. Let me know your thoughts! 😄

**Comment by koke** [comment]
Looks good to me as well, thanks for taking care of it
